rotkehlchen/api/v1/schemas.py

```diff
@@ -3694,3 +3701,54 @@ def make_calendar_query(
             identifiers=data['identifiers'],
         )
         return {'filter_query': filter_query}
+
+
+class CalendarReminderCommonEntrySchema(Schema):
+    AT_TIME_REGEX = re.compile(r'^(2[0-3]|[0-1][0-9]):[0-5][0-9]$')
```

Comment:
Mark it as constant. This should be a constant class variable shared by all inherited classes right? Not repeating in memory.
```suggestion
    AT_TIME_REGEX: Final = re.compile(r'^(2[0-3]|[0-1][0-9]):[0-5][0-9]$')
```